pyproject.toml

```
@@ -27,7 +27,7 @@ classifiers=[
 ]
 dependencies = [
 "numpy>=1.26.4,<2.0",
-"accelerate>=0.20.3,<0.40",
+"accelerate>=0.20.3,<0.34",
```

I think is clearer to explicitly set this dependency so we know that we are trying to install v0.33.0
```suggestion
"accelerate==0.33.0",
```